### ISSUE TITLE
Limit duplicate notifications sent to phone numbers

### ISSUE THREAD

**Issue by acthelemann** [COVID-19-electronic-health-system/Coronalert#38]
We want to limit the number of duplicate notifications sent to phone numbers. Instead of randomly picking from the notification list, which could result in the same notification sending consecutively, send the notifications one after the other.

This will require a new field in the MongoDB collection, something like `lastNotificationIndex`. When the next notification is sent, increment `lastNotificationIndex` and send the notification at that index.

There could be other ways to go about this, I'm open to suggestions.

**Comment by pavel-ilin** [comment]
Can work on this one

**Comment by pavel-ilin** [comment]
When I trying to connect to the mongoDB it tells me that my IP is not allowed to connect. Or may be my account don't have enough rights.

![](https://i.ibb.co/jZxhrkd/Screen-Shot-2020-04-29-at-12-10-11-PM.png)

**Comment by acthelemann** [comment]
Yeah, we need to add your IP to the whitelist. Give me your email I can add you to MongoDB
